update

```diff
@@ -39,8 +39,13 @@ fs.readdir(path.join(__dirname, '/bin'), function (err, files) {
     .map(function (pars) {
       const fileContent = readFile(pars.path)
       pars.longVersion = buildVersion(pars)
+      pars.sha256 = '0x' + ethUtil.sha256(fileContent).toString('hex')
       pars.keccak256 = '0x' + ethUtil.sha3(fileContent).toString('hex')
-      pars.urls = [ 'bzzr://' + swarmhash(fileContent).toString('hex') ]
+      pars.urls = [
+        'bzzr://' + swarmhash(fileContent).toString('hex'),
+        'https://ethereum.github.io/solc-bin/bin/' + pars.path,
+        'http://ethereum.github.io/solc-bin/bin' + pars.path
```

Comment:
Actually since we have `solc-bin.ethereum.org` as a redirect there, should we use that?

Reply:
Why do we need absolute links here? This file is just a summary of the other files in the same location.

Reply:
It would work if we consider the current approach with github pages, however I was thinking about having the "manifest" (`list.json`) available on swarm or ipfs and as such it would need absolute URLs.

We could include a relative path as an option in the URLs though.

Reply:
Actually if you use a swarm manifest, that would be an argument to use relative paths (which are then valid both inside swarm and inside github pages).

Reply:
Oh btw it is a URL and not a path. Not sure how to represent relative paths on URLs.

Reply:
We already have the `path` member, so actually no need to put them here at all.
We could add ipfs urls, though...

Reply:
I think we actually want to add the official `https://binaries.soliditylang.org/...` URL.

Reason: all the URLs we have in this array are pointing to a specific file and are not paths within a manifest/collection. If we publish `list.json` + the paths as a collection as well, for that use case the consumer can try the path, otherwise if `list.json` is just independently published on IPFS or swarm they can still opt to acquire the files via various means.